File: leetcode/1easy/이현진/2784.js
var isGood = function (nums) {
  let len = nums.length;
  nums.sort((a, b) => a - b);
  if (nums[len - 1] !== nums[len - 2]) return false;
  for (let i = 0; i < nums.length - 1; i++) {
    if (nums[i] != i + 1) return false;
  }
  return true;
};

//처음 생각한 방법 : max 값을 찾고 그 값을 제외한 애들은 한개이고 그 값은 두개인지 확인하기
//두번 째 방법 : 배열을 정렬한 후 맨 뒤에 있는 값을 두 번 나오는지, 나머지 값들은 한 번씩 나오는지 확인하기

File: leetcode/1easy/임강민/2784.py
# Check if Array is Good
<<<<<<< HEAD
class Leetcode.이재훈.이재훈.Solution:
=======
class Leetcode.이재훈.Solution:
>>>>>>> ab7f46f73d8f18bcd38c5cf06ecabae64f53a424
    def isGood(self, nums: List[int]) -> bool:
        flag = True
        nums.sort()
        for i in range(len(nums)):
            if not flag:
                break
            if i >= len(nums) - 2:
                if nums[i] != len(nums) - 1:
                    flag = False
            else:
                if nums[i] != i + 1:
                    flag = False
        return flag
